### ISSUE TITLE
View reviews by a reviewer

### ISSUE THREAD

**Issue by click2tman** [LamTechConsult/obizr#26]
The motivation behind this is to ensure that people viewing the a business detail page can see reviews of the business and should be able to see the details of the reviews and the profile of the reviewers with a summary of other reviews they have written.

Actions:

- While a user is looking at a business details page, they will see the reviews of that business below the details. 
- When they click on the review title/subject or body or block of text, they will be redirected to view the details of the review they clicked on.
- When they click on the reviewers photo to the left of the review listings, they should be redirected to a view displaying a summary of the reviewers profile and all the reviews written by that reviewer.
- While they are on the reviewers profile page and they click on a review title/subject or body or block of text , you should redirect the user to view the details of the review they clicked on.

**Business Details Page**
http://localhost:8100/#/app/businessDetails/31398 where 31398 is the node id of the business
![screen shot 2017-01-28 at 1 26 00 pm](https://cloud.githubusercontent.com/assets/1200007/22398955/1c6ca71a-e560-11e6-8fca-c6700c458af4.png)

**Reviewer Profile Page**
http://localhost:8100/#/app/reviewerProfile/771 where 771 is the uid of the reviewer
Recent reviews from a reviewer are located at http://slbr.sl/api/v1/slbiz/reviewer/%/reviews.json where % is the uid of the reviewer or revieweruid from the json feed
![screen shot 2017-01-28 at 1 33 32 pm](https://cloud.githubusercontent.com/assets/1200007/22398949/0a9bf400-e560-11e6-8869-fefc4ae8ef63.png)

**Review Details Page**
http://localhost:8100/#/app/reviewDetails/119 where 119 is the reviewid
![screen shot 2017-01-28 at 1 20 04 pm](https://cloud.githubusercontent.com/assets/1200007/22398983/962d1d6e-e560-11e6-9461-6f29503d55af.png)

**Comment by stevesolanki77** [comment]
I've made some changes.Please check if this is working.

**Comment by click2tman** [comment]
Ok. I will test this and provide feedback

**Comment by click2tman** [comment]
I tested this but its not working. I am not able to see reviews by a single reviewer only

**Comment by stevesolanki77** [comment]
I've made some changes... please check.. and should the add friend and follow need to do something? if so please mention .

**Comment by click2tman** [comment]
Yes. Add friend and follow are other issues in other milestones you will work on later. This view is still not working as desired. I still see reviews by other people when I am looking at http://localhost:8100/#/app/reviewerProfile/62254, 62254 is the uid of a new user I just created by signing up. on http://localhost:8100/#/app/reviewerProfile/[uid], it should be http://localhost:8100/#/app/reviewerProfile/[uidofthereviewer] not the person that is logged in. the reviews displayed should be only those written by the same reviewer.

**Comment by stevesolanki77** [comment]
please provide the username password of the new account so i can check.

and also please make some API changes to business details page. when using NodeResource for comments. the incoming results do not contain a reviwerId for each review.

the array object is like this
```
[
  {
    "cid": "134",
    "pid": "0",
    "nid": "40361",
    "uid": "1",
    "subject": "Aasdasdasd",
    "hostname": "144.121.74.123",
    "created": "1485932854",
    "changed": "1485932853",
    "status": "1",
    "thread": "01\/",
    "name": "tlamin",
    "mail": "",
    "homepage": "",
    "language": "und",
    "uuid": "8df0f660-6732-4971-b644-ec2e9382910f",
    "node_type": "comment_node_ltc_restaurant",
    "registered_name": "tlamin",
    "u_uid": "1",
    "signature": "",
    "signature_format": "full_html",
    "picture": "0",
    "new": 1,
    "comment_body": {
      "und": [
        {
          "value": "Adasdasdasdasd",
          "format": null,
          "safe_value": "Adasdasdasdasd"
        }
      ]
    },
    "field_ltc_biz_photos": [
      
    ],
    "field_ltc_biz_rating": {
      "und": [
        {
          "rating": "60",
          "target": null
        }
      ]
    },
    "_drafty_revision_requested": "FIELD_LOAD_CURRENT",
    "fb_post_graph": [
      
    ],
    "microdata": {
      "#attributes": {
        "itemscope": ""
      },
      "field_ltc_biz_photos": {
        "#attributes": [
          
        ]
      },
      "field_ltc_biz_rating": {
        "#attributes": [
          
        ],
        "average_rating": {
          "#attributes": [
            
          ]
        },
        "user_rating": {
          "#attributes": [
            
          ]
        },
        "rating_count": {
          "#attributes": [
            
          ]
        }
      },
      "comment_body": {
        "#attributes": [
          
        ]
      }
    },
    "$$hashKey": "object:323"
  }
]
```

**Comment by stevesolanki77** [comment]
as you can see the is no reviewId output. so i cant reference it.
